### DISCUSSION TITLE
Is it ok to store the uosc folder inside a dotfiles repository?

### DISCUSSION THREAD

**Discussion by barraIhsan** [tomasklaen/uosc#925] [Q&A] [answered]
So, I have a dotfiles repository that, well contains my dotfiles, including mpv scripts.
And inside there currently I have the ziggy binary, and other directory such as `char-conv`, `elements`, `intl`, `lib`, and `main.lua`.
The question, is it ok to store the ziggy binary file inside a public github repository? Or can I make like a git submodule to the actual ziggy repository (that contains the binary itself)? Or maybe include a "downloader" script on the dotfiles? any idea?

**Comment by hrxn** [comment, marked as answer]
> The question, is it ok to store the ziggy binary file inside a public github repository?

Yes, why not?
There are file size limitations, but this binary file won't hit them anytime..

**Reply by barraIhsan** [reply]
no, i mean with the licensing and stuff like that?

**Reply by hrxn** [reply]
Yes, if you don't make any changes to the license, why not?

**Reply by barraIhsan** [reply]
oh, ok then. Thanks!
